domains/scythemedia.json

```diff
@@ -3,7 +3,15 @@
 		"username": "ScytheMediaTheDev",
 		"email": "tswartz22@aol.com" 
 	},
-	"record": {
-		"MX": ["mx1.improvmx.com"] 
+	  "record": {
+    "A": [
+      "185.199.108.153"
+      "185.199.109.153"
+      "185.199.110.153"
+      "185.199.111.153"
+    ],
+    "MX": [
+	"mx.improvmx.com"
+		  ]	
```

Comment:
i think you need to add mx1.improvmx.com and mx2.improvmx.com, then add a TXT for improvMX to work